tests/queries/0_stateless/02156_storage_merge_prewhere.reference

```diff
@@ -1,6 +1,5 @@
 SELECT count()
 FROM t_02156_merge1
-PREWHERE k = 1
```

Comment:
Let's remove this. This test will be fixed here https://github.com/ClickHouse/ClickHouse/pull/33580.

Reply:
done.

Reply:
it isn't...

Reply:
It is ok now..